packages/smooth_app/lib/pages/preferences/user_preferences_contribute.dart

```diff
@@ -108,6 +108,12 @@ class UserPreferencesContribute extends AbstractUserPreferences {
               UserPreferencesListTile.getTintedIcon(Icons.open_in_new, context),
           externalLink: true,
         ),
+        _getListTile(
+            'Help improve Open Food Facts in your country',
+            () async => LaunchUrlHelper.launchURL(
+                'https://wiki.openfoodfacts.org/Country_Support_- ${userPreferences.userCountryCode ?? 'France'}'),
```

Comment:
It's not the way it works:
* you need to maintain a Map of country/wiki-link, as stated in the issue, for the same limited list of countries, with the correct values
* if the current country is not in the Map, you mustn't display that ListTile.

Reply:
Okay will do